rfcs/text/0000-data-citation-plan.md

```diff
@@ -0,0 +1,139 @@
+Note: Scientists are *already* citing such project based URLs in publications.
+
+See the "Unresolved Questions" section as to whether or not a formal DOI is required for Phase 1.
+
+### Phase 2
+This is designed to satisfy the data contributor requirements for User Story #5.
+This allows the Data Operations team to make a “Data Release” (i.e. a curated data set) where the data release itself and/or the projects contained within are citable.
+A Data Release must be immutable.
+The Data Browser must provide users with access to each Data Release (i.e. immutable version of data) in addition to the “live/latest” view.
+The Data Browser must be able to provide users with a means to download the data and metadata associated with a Data Release.
+Since the Matrix Service does not support the ability to process older versions of input files, the output files from the Matrix Service *must* be stored as part of the immutable data set.
+The Data Store "collections" API provides a suitable means for recording the contents of a Data Release.
+
+### Phase 3
+This is designed to satisfy the data consumer requirements for User Story #5.
+This also fully satisfies User Story #2 by allowing a data consumer to cite an arbitrary set of data which spans multiple projects.
+The data consumer must be able to create discrete versions of their selected data and the Data Browser must provide users access to each version.
+The data consumer must be able to share a citable link to any of the versions they have created.
+
+### Implementation notes for DOI support
+A DOI provides a link of the form https://doi.org/xxxx which resolves via the hosting [Registration Agency](https://www.doi.org/registration_agencies.html) or [open-access repository](https://en.wikipedia.org/wiki/Open-access_repository).
+A DOI is the most commonly accepted means for citing documents and data in scientific publications.
+
+It is acceptable for a DOI to resolve to an external website (e.g. BioStudies, Zenodo, etc.) which could then provide a URL to the project in the Data Browser as well as the ability to store point-in-time copies (i.e. versions) of relevant files such as the download manifest, metadata tsv, matrix output file, etc.
+The stored manifest file could then be used in the HCA CLI to download the exact versions of the data and metadata for that version of the project. It is a requirement that the DCP never deletes older versions of cited data files, except for the special case of retraction of unconsented data.
+
+#### Possible DOI Registration Agencies and open-access repositories that provide the ability to assign DOIs:
+  - [BioStudies](https://www.ebi.ac.uk/biostudies/)
+  - [Zenodo](https://en.wikipedia.org/wiki/Zenodo)
+  - [Figshare](https://en.wikipedia.org/wiki/Figshare)
+  - [Crossref](https://en.wikipedia.org/wiki/Crossref)
+  - HCA DCP assigns its own DOIs by becoming a member of the International DOI Foundation (IDF)
+
+#### DOI Versioning
```

Comment:
Agreed, and most (all?) do.

Reply:
I think given the post prefix text is entirely driven by the service there aren't any who couldn't support versioning, I am not sure if Biostudies can at the moment but it would be a feature we could request